release.sh

```diff
@@ -1,15 +1,13 @@
 # Before you release
-# - install pandoc on your system: apt-get install pandoc
-# - install pypandoc: pip install pypandoc
 # - install twine: pip install twine
-# - bump package version in setup.py
+# - bump package version in setup.cfg
 # - update the package version in the CHANGES file
 # - commit the changes to master and push
 
-PKG_NAME=$(python setup.py --name)
-PKG_VERSION=$(python setup.py --version)
+PKG_NAME=$(python read_setup_cfg.py --section metadata --key name)
+PKG_VERSION=$(python read_setup_cfg.py --section metadata --key version)
```

Comment:
It's kind of a bummer we have to write `read_setup_cfg.py`... I was browsing around and found this to extract the version:

```python
>>> from importlib import metadata
>>> metadata.version('pycobertura')
'2.1.0'
```

But I don't know if that would be available by the time we try to run this code (chicken-and-egg problem?)

Reply:
I also stumbled upon this about [single-sourcing the package version](https://packaging.python.org/en/latest/guides/single-sourcing-package-version/). Would it be relevant to what we're trying to achieve?

Reply:
I think actually even better would be the following:
```
from setuptools.config import read_configuration

conf_dict = read_configuration("/home/user/dev/package/setup.cfg")
```
Reference: https://setuptools.pypa.io/en/latest/setuptools.html.

I will test this locally and check in again.
Also with regards to the Python 3.7 problems we have (maybe it is the pip version, maybe it it is the python version that still needs a setup.py in editable mode).

Reply:
Unfortunately only supported from Python 3.8 onwards, but we can use `pkg_resources`. Python 3.7 is still supported for another 1 and a half years, so we should probably support it. This is also why we still need a empty setup.py with just an empty setup function call - although locally it works without